packages/clerk-js/src/ui/components/Checkout/CheckoutForm.tsx

```diff
@@ -40,14 +43,33 @@ export const CheckoutForm = ({
         })}
       >
         <LineItems.Root>
-          <LineItems.Group>
+          {showDowngradeInfo && (
+            <LineItems.Group expand>
+              <LineItems.Title description={'Your features will remain the end of your current subscription.'} />
+            </LineItems.Group>
+          )}
```

Comment:
This feels more like a notice verse a line item. Especially when there is no accompanying description.